README.md

```diff
@@ -30,6 +30,50 @@ latest JavaScript features; please also see [this wiki page](https://github.com/
 
 + Older browsers: https://mozilla.github.io/pdf.js/legacy/web/viewer.html
 
+## Setup Instructions (Platform-Specific)
```

Comment:
This section duplicates a lot of the information already found under the https://github.com/mozilla/pdf.js?tab=readme-ov-file#getting-the-code heading, which seems strange and potentially confusing.

Please add e.g. relevant sub-headings under that existing heading instead, rather than adding a new section here.